packages/cosmwasm/src/signingcosmwasmclient.ts

```diff
@@ -31,9 +34,9 @@ import {
 } from "./msgs";
 
 /**
- * Those fees are used by the higher level methods of SigningCosmWasmClient
+ * These fees are used by the higher level methods of SigningCosmWasmClient
  */
-export interface FeeTable {
+export interface CosmWasmFeeTable extends FeeTable {
```

Comment:
If we use extend here (assuming the CosmWasmClient can do everything the CosmosClient can), then we can remove the `send` field here, right?

Reply:
Yes. Do you think we have any need for the completely generic `FeeTable` (e.g. to support clients which turn off some Cosmos SDK functionality)?

Reply:
No, this is good for now.

I wanted to have CosmWasmClient extending CosmosClient but that does not work because there is no multi inheritance in JS and we need inheritance for SigningCosmWasmClient and SigningCosmosClient.

Reply:
OK, well we have `FeeTable` anyway. 😄